templates/welcome.php

```diff
@@ -0,0 +1,104 @@
+<?php
+\load_template(
+	\dirname( __FILE__ ) . '/admin-header.php',
+	true,
+	array(
+		'settings' => '',
+		'welcome' => 'active',
+	)
+);
+?>
+
+<div class="privacy-settings-body hide-if-no-js">
+	<h2><?php \esc_html_e( 'Welcome', 'activitypub' ); ?></h2>
+
+	<p><?php \esc_html_e( 'ActivityPub turns your blog into a federated social network. This means you can share and talk to everyone using the ActivityPub protocol, including users of Friendica, Pleroma and Mastodon.', 'activitypub' ); ?></p>
+	<p>
+		<?php
+		\printf(
+			// translators:
+			\esc_html__(
+				'People can follow you by using the username %1$s or the URL %2$s. Users who can not access this settings page will find their username on the %3$sEdit Profile%4$s page.',
+				'activitypub'
+			),
+			\sprintf(
+				'<code>%s</code>',
```

Comment:
```suggestion
				'<code>@%s</code>',
```

Should we prepend an `@` here? `@username@domain.blog`?